orbit/cmd/desktop/desktop.go

```diff
@@ -67,7 +68,9 @@ func main() {
 		}
 		rootCA := os.Getenv("FLEET_DESKTOP_FLEET_ROOT_CA")
 
-		client, err := service.NewDeviceClient(basePath, deviceToken, insecureSkipVerify, rootCA)
+		capabilities := []fleet.Capability{fleet.CapabilityTokenRotation}
```

Comment:
so, technically, the only one that needs to send  `CapabilityTokenRotation` is the Fleet Server, but I'm setting the value for Orbit, Desktop too in case the server needs to check this in the future.

Is this a good idea? should we establish guidelines. If the capabilities array is always going to be the same for everyone, maybe we should rename `fleet.ServerCapabilities` to something more generic and just use that?

Reply:
I would leave it generic as `[]Capability` if we think this won't be something that only Orbit/FD will use, but also the server.

I suppose there's a chance the server sends something different in return to an API based on the capabilities.

Reply:
Re sending the `CapabilityTokenRotation` capability from the client, my first thought is YAGNI, but if you think is going to be used in the near future, I'm ok with it.

Reply:
sounds good, I removed it for now.